lib/graphql/schema/relay_classic_mutation.rb

```diff
@@ -31,34 +35,14 @@ class RelayClassicMutation < GraphQL::Schema::Mutation
       def resolve_with_support(**inputs)
```

Comment:
The behavior has not changed, although a slight modification has been made due to the effect of cutting out input-related processing.